src/views/data.js

```diff
@@ -0,0 +1,99 @@
+const util = require('../lib/util');
+const messagesUtil = require('../lib/messages');
+
+const defaultSummary = { total: '--', time: '--' }
+
+function processType(type, indexedMessages) {
+    const messages = indexedMessages[type];
+    const result = { total: 0, time: 0 };
+    if (messages && messages.length) {
+        messages.forEach(function(message) {
+            result.total++;
+            result.time = parseInt(message.payload.duration);
```

Comment:
🤔 This overwrites `result.time` every time, a bit confusing what's happening here, maybe a comment?